grails-app/services/org/pih/warehouse/receiving/ReceiptService.groovy

```diff
@@ -248,6 +249,19 @@ class ReceiptService {
             }
         }
 
+        // Send notification email on completed receiving
```

Comment:
I feel like the save shipment should occur before the email is sent. Can you explain the reason for not doing that?